awx/main/models/base.py

```diff
@@ -61,7 +60,10 @@
     (PERM_INVENTORY_CHECK, _('Check')),
 ]
 
-CLOUD_INVENTORY_SOURCES = list(CLOUD_PROVIDERS) + ['scm']
+
+def discover_cloud_inventory_sources():
```

Comment:
I'd perhaps replace “discover” with “compute” in this function, since it calls the discovery internally but then augments whatever it returns. And double discovery in naming seems confusing.

Reply:
Thank you, that makes sense! I used your suggestion and have updated these to `compute_cloud_inventory_sources` via this commit: https://github.com/ansible/awx/pull/15537/commits/139b3d6798cd7b8b57efab56faa2b7ff6196b4ab